### PR TITLE
SPTCH-3493: Added aspectRatio option to window config.

### PR DESCRIPTION
**Pull request by nmoinvaz**

### CODE REVIEW

**Comment by steve-tucker** [comment]
Did you mean to leave this as a draft?

**Comment by nmoinvaz** [comment]
I will wait until the PR is merged in the client repo.

**Comment by nmoinvaz** [comment]
This is ready now.

**Review by steve-tucker — APPROVED** [review]

**Pull request merged**
